{"metadata":{"role":"symbol","fragments":[{"text":"SMIFormResponseErrorTypeTypeInternalServerError","kind":"identifier"}],"modules":[{"name":"SMIClientCore"}],"roleHeading":"Global Variable","title":"SMIFormResponseErrorTypeTypeInternalServerError","externalID":"c:@SMIFormResponseErrorTypeTypeInternalServerError","navigatorTitle":[{"text":"SMIFormResponseErrorTypeTypeInternalServerError","kind":"identifier"}],"symbolKind":"var"},"kind":"symbol","hierarchy":{"paths":[["doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore"]]},"schemaVersion":{"patch":0,"minor":3,"major":0},"sections":[],"abstract":[{"type":"text","text":"An internal service error was triggered preventing the submission of the form."}],"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"languages":["occ"],"tokens":[{"kind":"keyword","text":"extern"},{"kind":"text","text":" "},{"identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIFormResponseErrorType","kind":"typeIdentifier","text":"SMIFormResponseErrorType","preciseIdentifier":"c:SMIFormResponseErrorType.h@T@SMIFormResponseErrorType"},{"kind":"text","text":" "},{"kind":"keyword","text":"const"},{"kind":"text","text":" "},{"kind":"identifier","text":"SMIFormResponseErrorTypeTypeInternalServerError"},{"kind":"text","text":";"}]}],"kind":"declarations"}],"variants":[{"traits":[{"interfaceLanguage":"occ"}],"paths":["\/documentation\/smiclientcore\/smiformresponseerrortypetypeinternalservererror"]}],"identifier":{"interfaceLanguage":"occ","url":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIFormResponseErrorTypeTypeInternalServerError"},"references":{"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIFormResponseErrorType":{"kind":"symbol","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIFormResponseErrorType","type":"topic","abstract":[{"text":"An enum that defines ","type":"text"},{"code":"SMIFormResponseErrorType","type":"codeVoice"},{"text":" errors returned due to a failed form submission.","type":"text"},{"text":" ","type":"text"},{"text":"Possible values:","type":"text"}],"url":"\/documentation\/smiclientcore\/smiformresponseerrortype","role":"symbol","title":"SMIFormResponseErrorType","fragments":[{"kind":"identifier","text":"SMIFormResponseErrorType"}]},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore":{"identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","abstract":[],"type":"topic","role":"collection","url":"\/documentation\/smiclientcore","title":"SMIClientCore","kind":"symbol"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIFormResponseErrorTypeTypeInternalServerError":{"type":"topic","fragments":[{"kind":"identifier","text":"SMIFormResponseErrorTypeTypeInternalServerError"}],"kind":"symbol","url":"\/documentation\/smiclientcore\/smiformresponseerrortypetypeinternalservererror","title":"SMIFormResponseErrorTypeTypeInternalServerError","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIFormResponseErrorTypeTypeInternalServerError","abstract":[{"type":"text","text":"An internal service error was triggered preventing the submission of the form."}],"role":"symbol"}}}